.fixtures.yml

```diff
@@ -1,5 +1,6 @@
 fixtures:
   repositories:
     stdlib: https://github.com/puppetlabs/puppetlabs-stdlib
+    dism: https://github.com/puppetlabs/puppetlabs-dism
```

Comment:
Actually not required, it was a idea about downloading and installing via DISM since it has been reborn for Microsoft.